app/table/QueryTable.tsx

```diff
@@ -115,7 +112,6 @@ const makeQueryTable = <Item extends Record<string, unknown>>(
       <>
         <Table table={table} rowHeight={rowHeight} />
         <Pagination
-          inline={pagination === 'inline'}
```

Comment:
This was always false because `pagination` defaults to `"page"` and we never set the prop explicitly.